dallinger/command_line/mturk.py

```diff
@@ -0,0 +1,79 @@
+import click
+
+hit_id = click.argument(
+    "hit_id",
+    required=True,
+    type=str,
+)
+
+
+@click.group()
+@click.pass_context
+def mturk(ctx):
+    """Utility commands to interact with Mechanical Turk."""
+    pass
+
+
+@mturk.command("compensate")
```

Comment:
Just a note for now -- maybe in the final version, these commands can be defined programmatically so that we don't have to write it all out repeatedly for different recruiters?